src/inverseRobot.js

```diff
@@ -7,7 +7,25 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  let array = [];
+
+  // Change object into one array
+  Object.entries(robot).forEach(par => {
+    array = array.concat(par);
+  });
+
+  // check if array constains duplicates
+  const checkDuplicates = (new Set(array)).size !== array.length;
+
+  if (checkDuplicates) {
+    return null;
+  }
+
+  // make arrays from objects keys and swap them;
+  const fixRobot = Object.entries(robot).map(([key, value]) => [value, key]);
+
+  // change arrays into object and return
+  return Object.fromEntries(fixRobot);
```

Comment:
It's a working solution. But let's think about how to solve it easier.

1. you could create the empty object
2. iterate through the existing object using `for in` loop
3. check the value stored in the object by the current key. 
4. If you have a such key in the inversed object, then return null
5. If not, just set in the inversed key

One loop solution without any additional arrays.